src/amo/store.js

```diff
@@ -19,27 +19,29 @@ import search from 'core/reducers/search';
 import { middleware } from 'core/store';
 
 
+export const AMO_REDUCERS = {
```

Comment:
Heh, I feel like it's just how one does constants, so nothing other than convention.